src/about/donations/DonationsContainer.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './DonationsContainer.css'
+import Donors from './Donors'
+
+class DonationsContainer extends React.Component {
+
+  render(){
+    return (
+      <div className="container donations-container">
+        <h2>Donations</h2>
+        <p>Your support means a lot to us. With your help, we're able to accelerate our 
+        work on sharding, furthering our goal to scale the Ethereum blockchain for all. We have set 
+        up a multisig smart contract to accept donations.</p>
+        <a href="https://etherscan.io/address/0x9B984D5a03980D8dc0a24506c968465424c81DbE">
+          Address: 0x9B984D5a03980D8dc0a24506c968465424c81DbE
+        </a>
+        <p>Thank you to all who have contributed to Prysmatic Labs; we are eternally grateful. Check 
+        out some of our awesome patrons below.</p>
+        <Donors/>
```

Comment:
@rauljordan what do you think about the nomenclature here?

ECF and Ethereum Foundation are granters, i.e. the money given to us in the form of a grant has some stipulations and is not the same as a donation in the sense this is something we applied for and that we have to give the grant money back if we misuse it. 

Maybe we could split this up to have a Grant Awards section with link to the specific award annoucements and then also have a Donations section with links to the donator's web address and  donation transaction hash(s).

Reply:
@julientregoat Let's identify donors vs granters. It can be either by splitting these up into two sections or renaming the section to **Donors & Granters** with a bold title of **Donation** or **Grant  Award** for each respective party.

Also: Please add Aragon Nest since we were accepted into their program

Reply:
On it! Just gotta do this then PR should be good to go